nemo_text_processing/inverse_text_normalization/en/taggers/telephone.py

```diff
@@ -25,6 +31,17 @@
     PYNINI_AVAILABLE = False
 
 
+def get_serial_number(cardinal):
+    """
+    any alphanumerical character sequence with at least one number with length greater equal to 3
+    """
+    digit = pynini.compose(cardinal.graph_no_exception, NEMO_DIGIT)
+    character = digit | NEMO_ALPHA
+    sequence = character + pynini.closure(pynutil.delete(" ") + character, 2)
+    sequence = sequence @ (pynini.closure(NEMO_ALNUM) + NEMO_DIGIT + pynini.closure(NEMO_ALNUM))
+    return sequence.optimize()
+
```

Comment:
`thirty two a t` ->  `30 2at`. how about adding support for 10+ digits? and set the default to `32at`. There are pieces for `card numbers` that could be reused.

`seventy fifty five two a t` -> `7055 2at` (remove extra space?)

Reply:
resolved offline